src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,201 @@
+
+    :param colno: Column number
+    :param colname: Column name
+    :param entry: Column value
+    :return: Tuple int used to color the menu
+    """
+    if entry["is_default"]:
+        return 2, 0
+    return 3, 0
+
+
+def content_heading(obj: Any, screen_w: int) -> CursesLines:
+    """Create a heading for the setting entry showing.
+
+    :param obj: The content going to be shown
+    :param screen_w: The current screen width
+    :return: The heading
+    """
+    heading = []
+    string = obj["name"].replace("_", " ")
+    if obj["is_default"]:
+        string += f" (current/default: {obj['current_value']})"
+        color = 2
+    else:
+        string += f" (current: {obj['current_value']})  (default: {obj['default']})"
+        color = 3
+
+    string = string + (" " * (screen_w - len(string) + 1))
+
+    heading.append(
+        tuple(
+            [
+                CursesLinePart(
+                    column=0,
+                    string=string,
+                    color=color,
+                    decoration=curses.A_UNDERLINE,
+                ),
+            ],
+        ),
+    )
+    return tuple(heading)
+
+
+@actions.register
+class Action(ActionBase):
+    """The action class for the settings subcommand."""
+
+    KEGEX = r"^se(?:ttings)?$"
+
+    def __init__(self, args):
+        """Initialize the ``:settings`` action.
+
+        :param args: The current settings for the application
+        """
+        super().__init__(args=args, logger_name=__name__, name="settings")
+        self._settings: List[Dict]
+
+    def run(self, interaction: Interaction, app: AppPublic) -> None:
+        """Handle the ``settings`` subcommand in mode ``interactive``.
+
+        :param interaction: The interaction from the user
+        :param app: The app instance
+        :return: The pending :class:`~ansible_navigator.ui_framework.ui.Interaction` or
+            :data:`None`
+        """
+        self._logger.debug("settings requested")
+        self._prepare_to_run(app, interaction)
+
+        self._gather_settings()
+        self.steps.append(self._build_main_menu())
+
+        while True:
+            self._calling_app.update()
+            self._take_step()
+
+            if not self.steps:
+                break
+
+            if self.steps.current.name == "quit":
+                return self.steps.current
+
+        self._prepare_to_exit(interaction)
+        return None
+
+    def run_stdout(self) -> RunStdoutReturn:
+        """Handle settings in mode stdout.
+
+        :return: RunStdoutReturn
+        """
+        self._logger.debug("settings requested in stdout mode")
+        self._settings = transform_settings(self._args)
+        info_dump = human_dump(self._settings)
+        if isinstance(info_dump, str):
+            print(info_dump)
+            return RunStdoutReturn(message="", return_code=0)
+        return RunStdoutReturn(
+            message="Settings could not be retrieved, please log an issue.",
```

Comment:
Maybe include a link to the tracker?

Reply:
Downstream and upstream users follow a different process for logging issues, I'm not sure this is necessary.

Reply:
So maybe use two links? Is there a way to identify if it's a subscriber?

Reply:
No way I'm aware of.

Reply:
I don't think there is a current discovery path in navigator. you can probably build something to find that out but I think that is out of scope of this PR currently.

Reply:
Agreed. Resolving this thread.